arrows/qt/widgets/LoggerView.cxx

```diff
@@ -0,0 +1,71 @@
+// This file is part of KWIVER, and is distributed under the
+// OSI-approved BSD 3-Clause License. See top-level LICENSE file or
+// https://github.com/Kitware/kwiver/blob/master/LICENSE for details.
+
+#include "LoggerView.h"
+
+#include "ui_LoggerView.h"
+
+namespace kv = kwiver::vital;
+
+namespace kwiver {
+namespace arrows {
+namespace qt {
+
+//-----------------------------------------------------------------------------
+class LoggerViewPrivate
+{
+public:
+
+  Ui::LoggerView UI;
+};
+
+QTE_IMPLEMENT_D_FUNC(LoggerView)
+
+
+//-----------------------------------------------------------------------------
+LoggerView::LoggerView(
+  QWidget* parent, Qt::WindowFlags flags)
+  : QWidget{parent, flags}, d_ptr{new LoggerViewPrivate}
+{
+  QTE_D();
+
+  // Set up UI
+  d->UI.setupUi(this);
+
+  QFont font("Monospace", 10);
+  font.setStyleHint(QFont::TypeWriter);
+  d->UI.loggerText->document()->setDefaultFont(font);
+
+  QObject::connect(this, &LoggerView::logMessage,
+                   this, &LoggerView::appendMessage);
+}
+
+//-----------------------------------------------------------------------------
+LoggerView::~LoggerView()
+{
+}
+
+//-----------------------------------------------------------------------------
+void LoggerView::logHandler(kv::kwiver_logger::log_level_t level,
+                            std::string const& name,
+                            std::string const& msg,
+                            kv::logger_ns::location_info const& loc)
```

Comment:
***Very optional*** suggestion:
```suggestion
void LoggerView::logHandler(
  kv::kwiver_logger::log_level_t level, std::string const& name,
  std::string const& msg, kv::logger_ns::location_info const& loc)
```

Both are acceptable, I'm just noting this as an alternative.